lib/src/viam_sdk_impl.dart

```diff
@@ -18,9 +24,37 @@ class ViamImpl implements Viam {
   ViamMovementService? movementService;
   ViamSensorService? sensorService;
   DataService? _dataService;
+  late AppClient _appClient;
```

Comment:
(q) why are these `late`, but the above services are `null`able?

Reply:
The ones I added are `late`. The ones DoR added are optional because that's how they set it up